.github/workflows/workflow.yaml

```diff
@@ -192,3 +211,53 @@ jobs:
 
           if failed:
             sys.exit(1)
+
+  trivy:
+    name: Trivy Vulnerability Scan
```

Comment:
need to add info of working directory so that each run is unique, for instance `Trivy Vulnerability Scan in ${{ inputs.working-directory }}'

Reply:
done!